### PR TITLE
Auto-generate product org chart

### PR DESCRIPTION
**Pull request by jasonyavorsky**
This uses a generator to create the product org instead of having it be manually maintained on the page, using the same data that's used for the team page (so if you update things in one place it's reflected in both, and anywhere else this content is shown).

This is also a kind of proof of concept for having this data for the whole company, which we could use for example to replace https://handbook.sourcegraph.com/company/team/org_chart.

- Before: https://handbook.sourcegraph.com/product-engineering/product/product_org
- After: https://deploy-preview-895--sourcegraph-handbook.netlify.app/product-engineering/product/product_org

Instructions: https://deploy-preview-895--sourcegraph-handbook.netlify.app/handbook/editing/add-yourself-to-team-page#additional-steps-for-product

### CODE REVIEW

**Comment by jasonyavorsky** [comment]
For product specifically, what do you think @christinaforney ?

File: content/product/product_org.md [hunk 1]
```diff
@@ -1,29 +1,14 @@
...
-      - [Alicja Suska](../company/team/index.md#alicja-suska), Product Designer - [Frontend Platform](../engineering/enablement/frontend-platform/index.md), [Extensibility](../engineering/cloud/extensibility/index.md), [Code Insights](../engineering/code-graph/code-insights/index.md)
-      - [Quinn Keast](../company/team/index.md#quinn-keast), Product Designer - [Cloud SaaS](../engineering/cloud/saas/index.md) and [Search (Code monitoring & contexts)](../engineering/code-graph/search/index.md)
-      - [Sara Lee](../company/team/index.md#sara-lee), Product Designer - [Code intel](../engineering/code-graph/code-intelligence/index.md), [Extensibility](../engineering/cloud/extensibility/index.md)
-      - [Marisa Kanemoto](../company/team/index.md#marisa-kanemoto), Product Designer - [Search](../engineering/code-graph/search/index.md)
-  - [Technical Writing](roles/index.md#technical-writer)
-    - To be hired Head of Technical Writing. Christina acting lead until then.
+{{generator:reporting_structure.christina_forney}}
```
Comment (jyavorska) [inline content/product/product_org.md:7]:
```suggestion
{{generator:reporting_structure.christina_forney}}

<!-- the content for this page comes from team.yml and is built using the reports_to relationships -->
```

**Comment by jasonyavorsky** [comment]
Modify this to work with roles, just for product, and see if looks a little better.

**Comment by jasonyavorsky** [comment]
This now supports interim roles, and looks like this:

image

What do you think @christinaforney?

**Comment by jasonyavorsky** [comment]
@christinaforney @marybelzer I also added an update to the docs at https://deploy-preview-895--sourcegraph-handbook.netlify.app/handbook/editing/add-yourself-to-team-page#additional-steps-for-product describing the additional step that needs to be done when new team members are added if we go this route. What do you think? Is it too much?

File: content/handbook/editing/add-yourself-to-team-page.md [hunk 2]
```diff
@@ -31,3 +32,33 @@ marie_curie:
   pronunciation: https://forvo.com/word/marie_curie/
   description: I am a Polish and naturalized-French physicist and chemist who conducted pioneering research on radioactivity. I was the first woman to win a Nobel Prize, the first person and the only woman to win the Nobel Prize twice, and the only person to win the Nobel Prize in two scientific fields. My husband, Pierre Curie, was a co-winner on her first Nobel Prize, making us the first ever married couple to win the Nobel Prize and launching the Curie family legacy of five Nobel Prizes. I was, in 1906, the first woman to become a professor at the University of Paris.
 ```
+
+## Additional steps for product
```
Comment (jasonyavorsky) [inline content/handbook/editing/add-yourself-to-team-page.md]:
```suggestion
## Additional steps for Product
```

File: content/handbook/editing/add-yourself-to-team-page.md [hunk 3]
```diff
@@ -31,3 +32,33 @@ marie_curie:
...
+
+The product team is testing out a new feature to automatically generate our org chart. For now no other teams need to do anything different, but this eventually may roll out company-wide.
+
+If you have any trouble you can ask for help in #product-internal.
```
Comment (jasonyavorsky) [inline content/handbook/editing/add-yourself-to-team-page.md]:
```suggestion
If you have any trouble you can ask for help in #product-internal. If you are a new manager and want an interim manager role in your org chart, customize the same template below, but replace the root name, reporting line, and role description with whatever is appropriate.
```

File: content/handbook/editing/add-yourself-to-team-page.md [hunk 4]
```diff
@@ -31,3 +32,33 @@ marie_curie:
   pronunciation: https://forvo.com/word/marie_curie/
   description: I am a Polish and naturalized-French physicist and chemist who conducted pioneering research on radioactivity. I was the first woman to win a Nobel Prize, the first person and the only woman to win the Nobel Prize twice, and the only person to win the Nobel Prize in two scientific fields. My husband, Pierre Curie, was a co-winner on her first Nobel Prize, making us the first ever married couple to win the Nobel Prize and launching the Curie family legacy of five Nobel Prizes. I was, in 1906, the first woman to become a professor at the University of Paris.
 ```
+
+## Additional steps for Product
+
+The product team is testing out a new feature to automatically generate our org chart. For now no other teams need to do anything different, but this eventually may roll out company-wide.
+
+If you have any trouble you can ask for help in #product-internal. If you are a new manager and want an interim manager role in your org chart, customize the same template below, but replace the root name, reporting line, and role description with whatever is appropriate.
```
Comment (jasonyavorsky) [inline content/handbook/editing/add-yourself-to-team-page.md]:
```suggestion
If you have any trouble you can ask for help in #product-internal. If you are a manager and want an new interim manager role in your org chart, customize the same template below, but replace the root name, reporting line, and role description with whatever is appropriate.
```

**Comment by jasonyavorsky** [comment]
@nicksnyder curious what you think of this too, would something like this be useful for engineering, and maybe other departments at the company? Or too fancy/duplicates other sources of truth around open roles?

**Comment by nicksnyder** [comment]
Nice! Would love to use this in eng too!

for eng, I really like having the orgs/teams more visible like our current organization chart: https://handbook.sourcegraph.com/product-engineering/engineering/eng_org#current-organization

**Pull request merged**